modules/reindex/src/test/resources/rest-api-spec/test/reindex/90_remote.yaml

```diff
@@ -257,3 +257,35 @@
             match:
               text: test
   - match: {hits.total: 1}
+
+---
+"Reindex from with broken query":
```

Comment:
This didn't trigger the issue but I wrote it in an attempt to trigger it and I figure it is worth keeping anyway.